synapse/util/__init__.py

```diff
@@ -66,7 +67,7 @@ def looping_call(self, f, msec):
             f(function): The function to call repeatedly.
             msec(float): How long to wait between calls in milliseconds.
         """
-        call = task.LoopingCall(f)
+        call = task.LoopingCall(_log_exception_wrapper(f))
```

Comment:
is there a reason for doing this rather than adding an `errback` to the deferred returned by `call.start`?